gamemodes/terrortown/gamemode/server/sv_player_ext.lua

```diff
@@ -1259,12 +1259,8 @@ function plymeta:Give(weaponClassName, bNoAmmo)
 
 	local wep = plymeta_old_Give(self, weaponClassName, bNoAmmo or false)
 
-	timer.Simple(0, function()
-		if not IsValid(self) then return end
-
-		self.forcedPickup = false
-		self.forcedGive = false
-	end)
```

Comment:
Not really comfortable with removing the timer. As you are using the old give-function it seems like the whole code here could be just removed.
Or has forcedPickup and forcedGive and influence on the old give function?

Reply:
yes. The Give function internally calls the `GM:PlayerCanPickupWeapon` hook. The flags have to be set in order for it to work.